### PR TITLE
Editor: Make IOR configurable.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

This PR makes sure the `ior` property of `MeshPhysicalMaterial` can directly be configured and not just indirectly via `reflectivity`.

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing: https://rawcdn.githack.com/Mugen87/three.js/dev2/editor/index.html

File: editor/js/Sidebar.Material.js [hunk 1]
```diff
@@ -111,6 +111,11 @@ function SidebarMaterial( editor ) {
 	const materialReflectivity = new SidebarMaterialNumberProperty( editor, 'reflectivity', strings.getKey( 'sidebar/material/reflectivity' ) );
 	container.add( materialReflectivity );
 
+	// ior
+
+	const materialIOR = new SidebarMaterialNumberProperty( editor, 'ior', strings.getKey( 'sidebar/material/ior' ), [ 1, 2.333 ], 3 );
```
Comment (Mugen87) [inline editor/js/Sidebar.Material.js:116]: The [documentation](https://threejs.org/docs/index.html#api/en/materials/MeshPhysicalMaterial.ior) states that `ior` has a range of `[1, 2.333]`. However, there are glTF assets with a higher IOR value. E.g.  https://github.com/KhronosGroup/glTF-Sample-Assets/tree/main/Models/DispersionTest uses `2.42`. Should we allow a higher range in the editor e.g. `[ 1, 3 ]`?

Comment (gkjohnson) [reply editor/js/Sidebar.Material.js:116]: Do we know where that range come from? The [glTF docs](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_ior/README.md) don't seem to specify an upper bound:

> Valid values for ior are numbers greater than or equal to 1. In addition, a value of 0 is allowed.

But I agree that a max of something like 3 is good enough.

Comment (donmccurdy) [reply editor/js/Sidebar.Material.js:116]: glTF allows IOR values of 0, disallows values between 0 and 1 (exclusive), and allows 1 to infinity (inclusive).

https://github.com/KhronosGroup/glTF/blob/ae1a90baf009203760304f06ed5645402de9dc09/extensions/2.0/Khronos/KHR_materials_ior/schema/material.KHR_materials_ior.schema.json#L12-L20

2.42 is a typical value for diamonds according the KHR_materials_ior spec, but values >2 are unusual. Occasionally I see glTF files containing ior=0 or ior=1000+, these are not physically-plausible but are occasionally useful to give ...

> full weight to layer, i.e., the Fresnel term evaluates to 1 independent of the view or light direction. It is useful in combination with KHR_materials_specular to seamlessly support the specular-glossiness workflow.

Comment (Mugen87) [reply editor/js/Sidebar.Material.js:116]: Thanks for the feedback. I'll merge the PR with the current settings now so it aligns to the documentation. But it seems it's valid to reconsider the current range and think about adapting the glTF policy.

Comment (donmccurdy) [reply editor/js/Sidebar.Material.js:116]: If we need an example with higher IOR, a good way to test is to start with this model:

https://github.com/KhronosGroup/glTF-Sample-Assets/blob/main/Models/SpecGlossVsMetalRough/glTF-Binary/SpecGlossVsMetalRough.glb

It shows two bottles, one metal/rough and one spec/gloss. We don't support spec/gloss, so the latter will appear white in the editor:

Screenshot 2024-03-30 at 3 14 51 PM

If you convert the second bottle to metal/rough...

```
gltf-transform metalrough input.glb output.glb
```

... then the second bottle will include a material with ior=1000, and both bottles should appear identical:

Screenshot 2024-03-30 at 3 15 40 PM

**Pull request merged**